File: ARRAY/Sorting_Implementation/bubble_sort.py
def bubble_sort(arr):
    for i in arr:
        for j in arr:
            if arr[j] > arr[j + 1]:
                arr[j] , arr[j + 1] = arr[j +1] , arr[j]
arr = [5,2,32]
bubble_sort(arr)
print(arr)

File: ARRAY/Sorting_Implementation/bubble_sort.js
function bubble(arr){
    for(let i =0; i< arr.length - 1;i++){
        // watch for the length
        for(let j = 0; j < arr.length - 1 - i;j++){
            if (arr[j] > arr[j +1]){
                let temp = arr[j]
                arr[j] = arr[j + 1]
                arr[j + 1] = temp 
            }
        }
    }
}
arr = [2,2,2,1]
bubble(arr)
console.log(arr)